### PR TITLE
Danielgames

### PR DESCRIPTION
**Pull request by f8daniel**
Updates to Blackjack and Wordblocks

### Blackjack
Main changes:

1. **Improved game flow**
-    With 2+ players, players can now simultaneously select their wagers.
-    Removed all of the acknowledges, which slows game flow for player to read a notice and click okay

2.   **Enhanced Features**
-    Players can split their hand
-    Players can double down for a single card

3.  **Improved UI**
-    Included players' wagers and count (the hand score) in the playerboxes
-    Highlighted active playerbox to draw eye towards action
-    Improved messaging in the game log and with status updates
-    Adapted shim-notice from poker for an end of round summary of who won and who lost

4.   **Bug Fixing**
-    Playerboxes weren't being refreshed when player was eliminated -- fixed
-    misleading status update for dealer when player had a blackjack -- fixed

### Wordblocks
**Main Changes:**
-   New background, skin, color scheme, etc (per Kekya's design)
-   Introduced playerboxes for score keeping (per Kekya's design)
-   Improved responsive layout for visibility on smaller screens
-   reintroduced game log  
-   fixed bug for displaying the math on the last played word (now tracking individually per player
-   fixed bugs with initialization and undefined properties

### CODE REVIEW

File: lib/saito/ui/game-playerbox/game-playerbox.js [hunk 1]
```diff
@@ -145,14 +145,19 @@ class GamePlayerBox {
...
     * Groups all "opponent" playerboxes into a wrapper division
     */
-    groupOpponents(){
-      let html = `<div class="opponents" id="opponentbox"></div>`;
-      let oBox = this.app.browser.htmlToElement(html);
-
-      document.querySelector("#player-box-1").after(oBox); //Put new wrapper just after the player box 
-
+   groupOpponents(onlyOpponents = true){
+      let oBox = document.getElementById("opponentbox");
```
Comment (escapedcat) [inline lib/saito/ui/game-playerbox/game-playerbox.js:149]: Could be `const`?

File: lib/saito/ui/game-playerbox/game-playerbox.js [hunk 2]
```diff
@@ -145,14 +145,19 @@ class GamePlayerBox {
...
+
+      if (!oBox){
+        let html = `<div class="opponents" id="opponentbox"></div>`;
```
Comment (escapedcat) [inline lib/saito/ui/game-playerbox/game-playerbox.js:152]: Could be `const`?

File: mods/blackjack/web/style.css [hunk 3]
```diff
@@ -9,6 +9,34 @@ body {
   height: 100vh;
 }
 
+.player-box{
```
Comment (escapedcat) [inline mods/blackjack/web/style.css:12]: Could you run this file through https://prettier.io to format it correctly?

**Review by trevelyan — APPROVED** [review]
LGTM

File: mods/blackjack/web/style.css (hunk 3 above)
Comment (SankaD) [reply mods/blackjack/web/style.css:12]: prettier is already setup on the codebase. only on ts files though

**Pull request merged**